src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -295,4 +295,15 @@ std::array<double, 3> spin_vector(double spin_magnitude,
                                   const Scalar<DataVector>& ricci_scalar,
                                   const Scalar<DataVector>& spin_function,
                                   const YlmSpherepack& ylm) noexcept;
+
+/*!
+ * \ingroup SurfacesGroup
+ * \brief Irreducible mass of a 2D `Strahlkorper`.
+ *
+ * \details See Eqs. (15.38)
+ * of James B. Hartle's Gravity: An Introduction to Einstein's General
```

Comment:
Should this book be added to the bib file now that we have that set up? I found a bibtex on Google books:
```
@book{hartle2003gravity,
  title={Gravity: An Introduction to Einstein's General Relativity},
  author={Hartle, J.B.},
  isbn={9780805386622},
  lccn={2002151118},
  year={2003},
  publisher={Addison-Wesley}
}
```

Reply:
Nice idea! @jennifersanchez you can do this by adding the above bibtex to docs/References.bib and then replace the book info in your hop with `\cite hartle2003gravity`